File: sample/sample.js
// fake functions to be parsed by tree-sitter
// blehbleh
function purdue() {
    console.log("boilerup");
}

function login(user, pwd) {
    if ((user === "pete") && (pwd === "boilerUp123")) {
        console.log("success");
    } else {
        console.log("fail");
    }
}

function get_user() {
    return "pete";
}

function get_pwd() {
    return "boilerUp123";
}

File: sample/sample.py
# fake functions to be parsed by tree-sitter
#blehbleh
def purdue():
    print("boilerup")
def login(user, pwd):
    if ((user == "pete") and (pwd == "boilerUp123")):
        print("success")
    else:
        print("fail")

def get_user():
    return "pete"

def get_pwd():
    return "boilerUp123"